### ISSUE TITLE
Error installing cracklord-queued in Debian/Jessie

### ISSUE THREAD

**Issue by klingsor83** [jmmcatee/cracklord#109]
Hi,

I followed the instructions to install queueserver but the process fails when I execute
`apt-get install cracklord-queued`

I paste the error:
`Signature ok`
`subject=/O=Cracklord Fake Company/OU=Information Security/L=Chicago/ST=IL/C=US/CN=localhost`
`Getting CA Private Key`
`/var/lib/dpkg/info/cracklord-queued.postinst: 32: /var/lib/dpkg/info/cracklord-queued.postinst: initctl: not found`
`Failed to restart cracklord-queued.service: Unit cracklord-queued.service failed to load: No such file or directory.`
`dpkg: error al procesar el paquete cracklord-queued (--configure):`
 `el subproceso instalado el script post-installation devolvió el código de salida de error 6`
`Se encontraron errores al procesar:`
 `cracklord-queued`
`E: Sub-process /usr/bin/dpkg returned an error code (1)
`
I think that the package isn't generated correctly. In Ubuntu, the installation works without problem..

Also, I tried to install the QueueServer with one of the deb packages for Debian/Jessie in https://packagecloud.io/emperorcow/cracklord?filter=debs but it fails with the same error.

Regards

**Comment by jmmcatee** [comment]
So right now Cracklord only runs on Ubuntu 14.04. We are in the process of getting another packaging system to allow us to support more. We also only have an installer with an Upstart script so Debian will not work as a daemon. You should still be able to run it manually by executing "/usr/bin/cracklord-queued --conf="/etc/cracklord/queued.conf"". Let us know if that works as a short term solution.

**Comment by klingsor83** [comment]
Yes, it works running cracklord-queued manually. However, it would be great if we could install the package in Debian Jessie (without errors) to run cracklord as a daemon service with systemd.

**Comment by jmmcatee** [comment]
This has been added as a milestone 1.1 feature.
